dotrix_core/src/assets.rs

```diff
@@ -227,12 +230,44 @@ impl Assets {
     }
 }
 
+/// Reload assets and cleanup any assets that need some post process
+/// after `assets.remove`
+pub fn assets_reload(mut assets: Mut<Assets>, mut renderer: Mut<Renderer>) {
```

Comment:
Please add a new flag `hot_reload` to `Assets` and check it in the beginning of the system, so we can disable hot reloading.

Reply:
It is in

Reply:
We don´t need it to be run on every frame. Lets keep it now as is, maybe later we can introduce better solution.